client/lib/domains/index.js

```diff
@@ -299,6 +299,22 @@ function getAvailableTlds( query = {} ) {
 	return wpcom.undocumented().getAvailableTlds( query );
 }
 
+function getDomainTypeText( domain = {} ) {
+	switch ( domain.type ) {
+		case domainTypes.MAPPED:
+			return 'Mapped Domain';
+
+		case domainTypes.REGISTERED:
+			return 'Registered Domain';
+
+		case domainTypes.SITE_REDIRECT:
+			return 'Site Redirect';
+
+		case domainTypes.WPCOM:
+			return 'Wpcom Domain';
+	}
+}
```

Comment:
Shouldn't we add a `default` case to return a default value? Should we add a case to handle the [`TRANSFER`](https://github.com/Automattic/wp-calypso/blob/86304e1a0f303b038555d0960eb763537cddaaa6/client/lib/domains/constants.js#L8) domain type?

Reply:
done